### ISSUE TITLE
Pseudo selector does not support functions

### ISSUE THREAD

**Issue by eskimoblood** [cssinjs/react-jss#81]
I have the following style

```
const style = injectSheet({
  colored: {
    background: ({ theme }) => theme.color.primary.normal,
    '&:first-child': {
      paddingLeft: 5,
      background: ({ theme }) => theme.color.primary.light,
    },
  },
})(MyComponent)
```

While all elements get the normal color and the the first one get the additional padding, the first one don't get the different background

**Comment by iamstarkov** [comment]
#84 might be related

**Comment by kof** [comment]
Fixed, check out react-jss 6.0
